usagov/usagov-2021
# USAGOV 1366 Rename A11y and Lookup Scripts

## Jira Task
Renamed the scripts to better show their purpose


https://cm-jira.usa.gov/browse/USAGOV-1366

## Description

Renamed the scripts to better show their purpose. Also changed or indicated where their source names need to be changed. 

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

- For English: 
   - Go to http://localhost/elected-officials-results and click edit
   - Scroll to footer HTML 
   - Replace <script src="/themes/custom/usagov/scripts/lookup.js" defer></script> with 
                    <script src="/themes/custom/usagov/scripts/ceoResults.js" defer></script>

- For Spanish:
   -  Go to http://localhost/es/funcionarios-electos-resultados
   - Scroll to footer HTML 
   - Replace <script src="/themes/custom/usagov/scripts/lookup.js" defer></script> with 
                     <script src="/themes/custom/usagov/scripts/ceoResults.js" defer></script>

- Fill out CEO form for both languages. Make sure they are working as expected

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
